compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -518,8 +518,8 @@ static void addLowerToLLVMGPUPasses(OpPassManager &pm, bool useROCM) {
   pm.addPass(createConvertComplexToStandardPass());
 
   // Convert BF16 operations to occur as F32.
-  pm.addPass(createConvertBf16ArithToF32Pass());
-  pm.addPass(createConvertBf16ToUInt16BuffersPass());
+  // pm.addPass(IREE::Util::createPromoteArithBF16ToF32Pass());
```

Comment:
Do you mean to leave this in as commented?  Or simply eliminate?  if you want to keep it, please add a comment as to what this commented out code might help the reader understand / how it might be used later.

Reply:
Yes, ultimately this should go away. This is still a DRAFT PR.

Reply:
The alternative would be to add a command line option around this.
That could actually be useful for debugging purposes.
Up to you.

Reply:
Following yesterday's meeting, I think we should put these passes behind a flag.
Have this flag currently turned on (i.e., the passes run by default).
And flip the default for that flag when the LLVM changes land.
That way we can get this code in and we will see the performance boost in the benchmarks when we flip the flag.